tests/baselines/reference/errorSpanForUnclosedJsxTag.errors.txt

```diff
@@ -0,0 +1,27 @@
+tests/cases/compiler/errorSpanForUnclosedJsxTag.tsx(7,9): error TS17004: Cannot use JSX unless the '--jsx' flag is provided.
+tests/cases/compiler/errorSpanForUnclosedJsxTag.tsx(7,14): error TS17008: JSX element 'Foo.Bar' has no corresponding closing tag.
+tests/cases/compiler/errorSpanForUnclosedJsxTag.tsx(9,9): error TS17004: Cannot use JSX unless the '--jsx' flag is provided.
+tests/cases/compiler/errorSpanForUnclosedJsxTag.tsx(9,13): error TS17008: JSX element 'Baz' has no corresponding closing tag.
+tests/cases/compiler/errorSpanForUnclosedJsxTag.tsx(9,23): error TS1005: '</' expected.
+
+
+==== tests/cases/compiler/errorSpanForUnclosedJsxTag.tsx (5 errors) ====
+    let Foo = {
+      Bar() {}
+    }
+    
+    let Baz = () => {}
+    
+    let x = <    Foo.Bar >Hello
+            ~~~~~~~~~~~~~~
+!!! error TS17004: Cannot use JSX unless the '--jsx' flag is provided.
+                 ~~~~~~~
+!!! error TS17008: JSX element 'Foo.Bar' has no corresponding closing tag.
+    
+    let y = <   Baz >Hello
+            ~~~~~~~~~
+!!! error TS17004: Cannot use JSX unless the '--jsx' flag is provided.

```

Comment:
You can get rid of these errors by adding `// @jsx: react` at the top of `tests/cases/compiler/errorSpanForUnclosedJsxTag.tsx`

Reply:
Thanks for helping me sort this out.